apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -544,6 +562,10 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             "allow_feedback_badges": not all(
                 block.get("badge_type", "") == "" for block in student_submission
             ),
+            "block_layout": "pl-block-horizontal" if inline else "",
```

Comment:
Can the values for the keys `"block_layout"` and `"dropzone_layout"` be read from a hardcoded dictionary (instead of if statements like now)? The mapping logic seems to be the same for both, so reading the values from a dict will clean up the new code.

Reply:
Personally I don't see a need for this. The if statements are quite simple as it is. Factoring it out to a dictionary means you have to go look somewhere else in the code to see what its doing. 

I don't have particularly strong feelings for or against though, I'm cool with whatever.

Reply:
I might also lean towards keeping it as is. Though I'd agree Python's ternaries are less than pretty at times I do think its nice to just have the values front and center.

Reply:
It's less about reducing the amount of logic and more about reducing the number of string literals in the code. If changed, it's true you can't see the values directly on the same line, but you're relying on having copied the same strings correctly in multiple places.